docs/source/guide/repository-service-tuf-api/api/swagger.json

```diff
@@ -381,6 +381,39 @@
                 ]
             }
         },
+        "/api/v1/targets/publish/": {
+            "post": {
+                "tags": [
+                    "v1",
+                    "v1"
```

Comment:
Why there are two tags?

Reply:
I see the same pattern in other places as well in the swagger.json, so it's not an isolated thing.